recipes/log4cxx/all/patches/0002-fix-expat-target.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/CMakeLists.txt
```

Comment:
this patch can be avoided with set_property

Reply:
Can you tell me how? I tried:
```
  tc = CMakeDeps(self)
  if Version(self.version) < "0.13.0":
      tc.set_property("expat::expat", "cmake_target_name", "EXPAT::EXPAT")
  tc.generate()
```

Reply:
You need to use the Conan name of the recipe or its components, not the default cmake target